client/src/ts/dd_main.ts

```diff
@@ -207,6 +214,7 @@ ws.onmessage = (event) => {
     case canDefs.CanMessage.CAN_MESSAGE_MOTOR_CONTROLLER_VC:
       const power = msg.data.mc_voltage_1 * msg.data.mc_current_1 +
           msg.data.mc_voltage_2 * msg.data.mc_current_2;
+      consumptionReadout.value(power);
```

Comment:
Good catch! Probably makes more sense to have it here.

Just FYI, apparently the current value from the WaveSculptors is known to be somewhat inaccurate (at least for the 22s, we never really did any testing on the 20s, but I assume the calculation is performed the same way), so you might want to check this source used for calculation at some point before going to the race.

Can you create a ticket to follow up on :point_up: Basically outline that we should do a simple comparison the reported current against a measured current and quantify the error (and maybe fit a curve across a few datapoints)? Or at the very least sanity check against the power measured by the BMS. It's kind of low priority since the data should be transmitted anyways, but just have it in the backlog so we don't forget about it.

Reply:
Done, see [here](https://uwmidsun.atlassian.net/browse/ELEC-595)